R/PredictionSurv.R

```diff
@@ -148,9 +158,16 @@ PredictionSurv = R6Class("PredictionSurv",
         surv
       }
     },
-    .distrify_survmatrix = function(x) {
```

Comment:
This with `x` a survival array won't return the 3rd dimension as `mean`. But if you include the `which.curve = mean` it doesn't work as well. So only way I found was to to `Arrdist$new()`